app/models/policies/further_education_payments/admin_provider_verification_task_presenter.rb

```diff
@@ -115,7 +293,89 @@ def claimant_answer(assertion)
+          value = claim.eligibility.provider_verification_contract_covers_full_academic_year
+          value ? "Yes" : "No"
+        when "teaching_hours_per_week_next_term"
+          # Show provider's answer for variable hours next term teaching hours
+          teaching_hours_display_value(claim.eligibility.provider_verification_teaching_hours_per_week_next_term)
+        when "teaching_qualification"
+          # Show actual teaching qualification instead of Yes/No
+          teaching_qualification_display_value(claim.eligibility.provider_verification_teaching_qualification)
+        when "performance_measures", "disciplinary_action"
+          # Show inverted logic for performance/disciplinary (Yes means subject to, No means not subject to)
+          provider_performance_disciplinary_answer(key)
+        else
+          assertion["outcome"] ? "Yes" : "No"
+        end
+      end
+
+      private
+
+      def contract_type_display_value(contract_type)
+        case contract_type
+        when "permanent"
+          "Permanent"
+        when "fixed_term"
+          "Fixed term"
+        when "variable_hours"
+          "Variable hours"
+        when "employed_by_another_organisation"
+          "Employed by another organisation (for example, an agency or contractor)"
+        else
+          contract_type&.humanize || "Not provided"
+        end
+      end
+
+      def teaching_hours_display_value(hours)
+        case hours
+        when "20_or_more_hours_per_week"
+          "20 hours or more each week"
+        when "12_to_20_hours_per_week"
+          "12 hours to 20 hours each week"
+        when "2_and_a_half_to_12_hours_per_week"
+          "2.5 to 12 hours each week"
+        when "fewer_than_2_and_a_half_hours_per_week"
+          "Fewer than 2.5 hours each week"
+        else
+          hours&.humanize || "Not provided"
+        end
+      end
+
+      def teaching_qualification_display_value(qualification)
+        case qualification
+        when "yes"
+          "Yes"
+        when "not_yet"
+          "Not yet, I am currently enrolled on one and working towards completing it"
+        when "no_but_planned"
+          "No, but I plan to enrol on one in the next 12 months"
+        when "no_not_planned"
+          "No, and I do not plan to enrol on one in the next 12 months"
+        else
+          qualification&.humanize || "Not provided"
+        end
+      end
+
+      def provider_performance_disciplinary_answer(field)
+        eligibility = claim.eligibility
+        case field
+        when "performance_measures"
+          # Provider field is "subject to performance measures" - show Yes if true, No if false
+          eligibility.provider_verification_performance_measures ? "Yes" : "No"
+        when "disciplinary_action"
+          # Provider field is "subject to disciplinary action" - show Yes if true, No if false
+          eligibility.provider_verification_disciplinary_action ? "Yes" : "No"
+        end
```

Comment:
I think we want to use a separate task for year 2 FE claims so we only need to do the branching in one place (the decision on which task to render)